src/components/custom-field-input-number/custom-field-input-number.jsx

```diff
@@ -32,14 +32,19 @@ export default function CustomFieldInputNumber(props) {
   useEffect(() => {
     if (!inputRef.current) return;
 
-    setInvalid(!inputRef.current.validity.valid);
+    setInvalid(!inputRef.current.validity.valid || props.error);
   });
 
+  useEffect(() => {
```

Comment:
This was necessary to ensure that `invalid` updated along with `props.error`, but I was unsure how to test it. Do any of you have ideas on how to do just that?

Reply:
I was wondering whether `useState` allowed for a new props. This makes sense. 

There is a `rerender` method in `@testing-library/react` somewhere: https://testing-library.com/docs/react-testing-library/api#rerender